.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,32 +1,50 @@
-# EASI-0000, EASI-0000
-<!-- Follow the pattern of Parent issue, Child issue, if appropriate -->
-
-## Changes and Description
-
-- Change 1
-- Change 2
+<!--
+REQUIRED
+    Ensure that your PR title has the relevant Jira ticket number(s) in the title.
+    Follow this pattern: [EASI-1234] [EASI-4567] Title of the PR.
+    Use [NOREF] in place of a ticket number when there's no associated Jira ticket.
+    The heading below should just have the ticket numbers (for easy linking in Jira)
+-->
+# EASI-0000
+
+## Description
+<!--
+REQUIRED
+    Provide details as to what the PR aims to accomplish
+    Be as descriptive as you can, and include any relevant information that will help the reviewer understand the scope of the changes
+    Include screenshots or screen recordings to assist in reviewing if possible.
+-->
 
-<!-- Put a description here! -->
 
 ## How to test this change
-
 <!--
-    Add any steps or code to run in this section to help others run your code:
-
-    ```sh
-    echo "Code goes here"
-    ```
---->
-
-## PR Author Review Checklist
+REQUIRED
+    Add instructions on how to test the changes in this PR
+    This can be a list of steps to reproduce a bug, or a list of steps to verify a feature in the application
+    Include any example shell commands or postman requests that reviewers can run to test the changes
```

Comment:
Maybe we could add some language to include sql for test data if applicable? If the change is a major migration, the developer probably has made test data (that can be exported as sql). I think it can go a long way to simplify review.

Reply:
I like it! b43b0116